libtransact/src/protocol/transaction.rs

```diff
@@ -64,6 +64,33 @@ pub struct TransactionHeader {
 }
 
 impl TransactionHeader {
+    #[allow(clippy::too_many_arguments)]
+    pub fn new(
```

Comment:
Shouldn't there be a builder for this?

Reply:
Definitely skipped right over the TransactionBuilder when doing this, removed this method and changed the test to use TransactionBuilder instead